daemon/src/collab_settlement_taker.rs

```diff
@@ -0,0 +1,194 @@
+use crate::address_map::Stopping;
+use crate::model::cfd::{
+    Cfd, CollaborativeSettlement, OrderId, SettlementKind, SettlementProposal,
+};
+use crate::model::Price;
+use crate::{connection, projection, wire};
+use anyhow::{Context, Result};
+use async_trait::async_trait;
+use xtra::prelude::MessageChannel;
+use xtra_productivity::xtra_productivity;
+
+pub struct Actor {
+    cfd: Cfd,
+    projection: xtra::Address<projection::Actor>,
```

Comment:
:question: Should we talk directly to the `projection::Actor` in the other setup actors then?

Reply:
In this case I would say yes because the `taker_cfd::Actor` doesn't actually care about the ongoing settlement state other than to forward it to the `projection::Actor`.